rcl/include/rcl/service.h

```diff
@@ -252,7 +256,22 @@ rcl_take_request_with_info(
   rmw_service_info_t * request_header,
   void * ros_request);
 
-/// backwards compatibility version that takes a request_id only
+/// Backwards compatibility function to ake a pending ROS request using a rcl service.
```

Comment:
```suggestion
/// Backwards compatibility function to take a pending ROS request using a rcl service.
```

Reply:
is this function only used for backwards compatibility?
It seems to me that we're actually using it ...

Reply:
Fixed the typo in 2206ac8 .